standard/schema/utils/README.md

```diff
@@ -14,26 +14,4 @@ This can be created by running:
 
 There are tests set-up to ensure this has happened. 
 
-### make_field_definitions.py
```

Comment:
Since `make_field_definitions.py` is still used by `build_docs.sh`, should we add these docs to its file, if we're removing them from the readme?